scripts/quickstart.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
Do we need that now?

Reply:
I think it's still nice to have for a quick spinup for manual testing.

Reply:
ok